Main.java

```diff
@@ -9,7 +10,25 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // TODO code application logic here
+  Scanner input = new Scanner(System.in);
+
+  System.out.println("Please enter in the x coordinate of the first point");
+  
+  int X1 = input.nextInt();
+  
+  System.out.println("Please enter in the y coordinate of the first point");
+ 
+  int Y1 = input.nextInt();
+
+   System.out.println("Please enter in the x coordinate of the second point");
+  int X2 = input.nextInt();
+
+    System.out.println("Please enter in the y coordinate of the second point");
+
+  int Y2 = input.nextInt();
+ 
+  //declare and calculate
+  System.out.println("The slope of your line would be " + (Y2 -Y1)/(X2 - X1) );

```

Comment:
Can the x and y values have decimal places in them? They should probably be doubles instead of ints. You can get a little bit of a hint to this if you look at the sample. The slope says "2.0". Also, try not to do math inside of a print statement. Instead, place it in a variable and use that variable in the print statement. Finally, don't forget to comment your code!